portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,63 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const facts =
+      ['I have a tabby cat named Poppy.', 'I was in a drumline.', 'I\'m part of Y Fashion House at Yale.', 'I love to paint.', 'I am from Nashville.'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('message-container');
+  factContainer.innerText = "You found a hidden fact! \n \n" + fact + "\n \n Reload the page, and this fact will disappear";
 }
+
+// adds start from where
+var bar1 = new ProgressBar.Line(htmlBar, {
+  strokeWidth: 4,
+  easing: 'easeInOut',
+  duration: 1400,
+  color: '#FFEA82',
+  trailColor: '#eee',
+  trailWidth: 1,
+  svgStyle: {width: '100%', height: '100%'}
+});
+
+bar1.animate(1.0); 
+
+// the following is prob not needed
+//
+// const spans = document.querySelectorAll('h1 span')
+// spans.forEach(span => span.addEventListener ('mouseover',function(e){
+//   span.classList.add('animated', 'rubberBand')
+// }))
+//
+// spans.forEach(span => span.addEventListener ('mouseout',function(e){
+//   span.classList.remove('animated', 'rubberBand')
+// }))
+//
+// const htmlBar = document.querySelector('.bar-html')
+// const jsBar = document.querySelector('.bar-javascript')
+// const pythonBar = document.querySelector('.bar-python')
+// const javaBar = document.querySelector('.bar-java')
+//
+// var t1 = new TimelineLite()
+//
+// t1.fromTo(htmlBar, .75, {width: `calc(0% - 6px)`}, {width: `calc(90% - 6px)`, ease: Power4.easeOut})
+//   .fromTo(jslBar, .75, {width: `calc(0% - 6px)`}, {width: `calc(90% - 6px)`, ease: Power4.easeOut})
+//   .fromTo(pythonlBar, .75, {width: `calc(0% - 6px)`}, {width: `calc(90% - 6px)`, ease: Power4.easeOut})
+//   .fromTo(javalBar, .75, {width: `calc(0% - 6px)`}, {width: `calc(90% - 6px)`, ease: Power4.easeOut})
+//
+//   const contoller = new ScrollMagic.Controller()
+//   const scene = new ScrollMagic.Scene({
+//     triggerElement: '.skills',
+//     triggerHook: 0
+//   })
+//   .setTween(t1)
+//   .addTo(controller)
+
+  // const showRequiredCategory = event => {
+  //   const getId = event.id
+  //   const link = documnet.querySelectorAll()
+  //
+  // }
```

Comment:
Can you remove this before merging if it isn't needed. If you need it later, you can always find it again in your commit history.